tests/backend/unit/test_config.py

```
@@ -0,0 +1,924 @@
+        
+        Example:
+            monkeypatch.setenv("mobius_port", "7777")
+            monkeypatch.setenv("MOBIUS_HOST", "0.0.0.0")
+            settings = Settings()
+            assert settings.port == 7777
+            assert settings.host == "0.0.0.0"
+        
+        Generated by CodeRabbit
+        """
+        monkeypatch.setenv("mobius_port", "7777")
+        monkeypatch.setenv("MOBIUS_HOST", "0.0.0.0")
```

Ruff (0.11.9)

Possible binding to all interfaces

(S104)